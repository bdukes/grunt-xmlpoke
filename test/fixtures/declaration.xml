<?xml version="1.0" encoding="UTF-8"?>
<widget param1="12">
    <title>I want to be changed</title>
</widget>
