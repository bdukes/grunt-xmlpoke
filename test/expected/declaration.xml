<?xml version="1.0" encoding="UTF-8"?>
<widget param1="12">
    <title>New Title</title>
</widget>